expanalysis/experiments/jspsych_processing.py

```diff
@@ -877,6 +878,9 @@ def calculate_auc(data):
     dvs['min_rss_small'] = {'value': calculate_discount_rate(df_small)['min_rss'], 'valence': 'Neg'}
     dvs['min_rss_medium'] = {'value': calculate_discount_rate(df_medium)['min_rss'], 'valence': 'Neg'}
     dvs['min_rss_large'] = {'value': calculate_discount_rate(df_large)['min_rss'], 'valence': 'Neg'}
+    dvs['num_trials_small'] = {'value': calculate_discount_rate(df_small)['num_trials'], 'valence': 'Pos'}
+    dvs['num_trials_medium'] = {'value': calculate_discount_rate(df_medium)['num_trials'], 'valence': 'Pos'}
+    dvs['num_trials_large'] = {'value': calculate_discount_rate(df_large)['num_trials'], 'valence': 'Pos'}
```

Comment:
You are calling "calculate_discount_rate" three times per data-set. Just call it once and then reference the three variables ('discount_rate', 'min_rss' and 'num_trials') in the dvs